lib/harness/renderer/utils.ex

```diff
@@ -76,4 +79,8 @@ defmodule Harness.Renderer.Utils do
   defp join_compositions([kind_a, kind_b, kind_c]) do
     "#{kind_a}, #{kind_b}, and #{kind_c}"
   end
+
+  defp join_compositions([kind_a, kind_b, kind_c, kind_d]) do
+    "#{kind_a}, #{kind_b}, #{kind_c}, and #{kind_d}"
+  end
```

Comment:
I think this'll be useful if it captures 3+ elements instead of one clause for 3 and one clause for 4

```elixir
defp join_compositions(composition) do
  {first_elements, final_element} = Enum.split(compositions, -1)

  Enum.join(first_elements, ", ") <> ", and " <> final_element
end
```

not that I can really imagine adding more file types but /shrug

Reply:
went for it. tyty